src/components/HeaderApp.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="wrapper-header">
+      <div class="header">
+        <router-link class="header__logo" to="/">
+          <img src="@/assets/logo.svg" alt="logo">
+        </router-link>
+      <div class="header__nav">
+        <v-btn to="/raiting-movies" x-small class="header__nav__btn">Raiting films</v-btn>
+        <v-btn to="/some-movies" x-small class="header__nav__btn">Some films</v-btn>
+      </div>
+    </div>
+    </div>
+</template>
```

Comment:
This code has already been reviewed. Also, it's a HEADER component. But Where is a LOADER component?

It looks like rebase needed main into the current branch